client/lib/gsuite/new-users.ts

```diff
@@ -160,30 +173,77 @@ const validateNewUserMailboxIsUnique = (
 /*
  * Adds a duplicate error to each mailBox with a duplicate mailbox
  */
-const validateNewUsersAreUnique = ( users: GSuiteNewUser[] ) => {
+const validateNewUsersAreUnique = ( users: GSuiteNewUser[] ): GSuiteNewUser[] => {
 	const mailboxesByCount: { [ mailbox: string ]: number } = countBy(
 		users.map( ( { mailBox: { value: mailBox } } ) => mailBox )
 	);
 
-	return users.map( ( { uuid, domain, mailBox, firstName, lastName } ) => ( {
+	return users.map( ( { uuid, domain, mailBox, firstName, lastName, password } ) => ( {
 		uuid,
 		firstName,
 		lastName,
 		domain,
 		mailBox: validateNewUserMailboxIsUnique( mailBox, mailboxesByCount ),
+		password,
 	} ) );
 };
 
 /*
- * Run all validations on a user:
- * domain - required
- * mailBox - required, validEmailCharacters, valid overall email
- * firstName - required, less than sixty characters
- * lastName - required, less than sixty characters
+ * Validates the specified password field.
+ *
+ * @see https://support.google.com/accounts/answer/32040 for requirements
+ */
+const validatePasswordField = ( { value, error }: GSuiteNewUserField ): GSuiteNewUserField => {
+	if ( ! error && 8 > value.length ) {
```

Comment:
Why is the min-length 8 instead of 12 as proposed by https://support.google.com/accounts/answer/32040?

Reply:
Because that's what the API documentation [mentioned](https://developers.google.com/admin-sdk/directory/v1/reference/users/insert). I'll update it to 12.